doc/de/weechat_user.de.adoc

```diff
@@ -839,6 +839,8 @@ andere Items die zur Verfügung stehen (die aber nicht standardmäßig in einer
 | spell_dict                   | `de,en`                  | zeigt an welche Wörterbücher für die Rechtschreibung im aktuellen Buffer genutzt werden.
 | spell_suggest                | `Glück,Glocke,Block`     | Vorschläge für ein falsch geschriebenes Wort.
 | window_number                | `2`                      | Nummer des aktuellen Fensters.
+// TRANSLATION MISSING
+| tls_version                  | `TLS1.3`                 | TLS version in use for current IRC server.
```

Comment:
Items should be sorted alphabetically in this table (same remark for other languages).

Reply:
Fixed, thanks for taking another look.